sdk/provisioning/ci.data.yml

```diff
@@ -11,6 +11,7 @@ trigger:
     - sdk/provisioning/ci.data.yml
     - sdk/provisioning/Azure.Provisioning
     - sdk/provisioning/Azure.Provisioning.AppConfiguration
+    - sdk/provisioning/Azure.Provisioning.CloudMachine
```

Comment:
Can you do me a favor and move this into `ci.compute.yml` instead?  I'm trying to keep the pipelines balanced because I'm worried about blowing out azdo again with how big this one is getting.

Reply:
(Or consider adding your own release pipeline since I don't think this will be going out with everything else.)

Reply:
moved